lib/plugins/aws/customResources/resources/eventBridge/lib/utils.test.js

```diff
@@ -0,0 +1,23 @@
+'use strict';
+
+// eslint-disable-next-line import/no-extraneous-dependencies
```

Comment:
Let's not rely on lint silencers. If ESLint for some reason complains about that, let's find out why, and address the problem

Reply:
@medikoo, I fixed it. The reason is `lib/plugins/aws/customResources/resources` contains its own package.json file, which only require `aws-sdk`. Added `chai` as a dev dependency in this package resolves the issue.

However, I noticed that the generated `node_modules` directory within this sub-folder causes eslint to fail (it is linting dependencies files) and I therefore had to add an `.eslintignore` file at project root to ensure all `node_modules` subdirectory are ignored - [root `node_modules` is ignored by default](https://eslint.org/docs/user-guide/configuring#eslintignore). Doing so, I noticed quite a few linting errors existing in `lib/plugins/create/templates/` directories (do you have any knowledge of any existing configuration that was overruled by the new `.eslintignore` file and that was preventing linting of this folder ?). I ignored this path as well for the time being as it is outside of the scope of this fix. Would you like me to create an issue related to this for later ?